src/EnergyPlus/DataVectorTypes.hh

```diff
@@ -88,6 +88,13 @@ namespace DataVectorTypes {
 	typedef  ObjexxFCL::Vector3< Real64 >  Vector;
 	typedef  ObjexxFCL::Vector2< Real64 >  Vector_2d;
 
+	struct Vector2dCount: Vector_2d {
+		int count;
+		Vector2dCount():
+			count( 0 )
+		{}
+	};
+
```

Comment:
Someone must've hacked my account when making that comment, cause I'd never miss something like that.  

:smirk: 

Got it, thanks!